### DISCUSSION TITLE
Advanced filter improvement idea

### DISCUSSION THREAD

**Discussion by dfaruque** [serenity-is/Serenity#6281] [Serenity Ideas]
Nice to have this kind of filtering in serenity framework.
https://www.signumsoftware.com/en/Demo
https://southwindsqlserver.azurewebsites.net/find/Order

**Comment by VictorTomaili** [comment]
this kind? 

![image](https://user-images.githubusercontent.com/2023440/154839928-7e9f2771-d568-4cb3-93ca-98cfd78b6186.png)

**Reply by dfaruque** [reply]
Thank you.
I know about this in serenity and used it. but I point that the filtering with the related entity properties. they managed it somehow. so I have shared this as an idea.

**Reply by dfaruque** [reply]
even they support filtering by some property about property. for example: for a string field, it can filter by length etc.

![image](https://user-images.githubusercontent.com/16621563/154842464-bd025c91-0dcd-4ef2-809a-2eb56b85ce28.png)
